### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by lookfirst**
Just some small cleanup commits as I'm reading the code.

### CODE REVIEW

File: libpoolprotocols/stratum/EthStratumClient.cpp [hunk 1]
```diff
@@ -277,11 +277,11 @@ void EthStratumClient::disconnect_finalize() {
 }
 
 void EthStratumClient::resolve_handler(const boost::system::error_code& ec, tcp::resolver::iterator i)
-{	
+{
+    dev::setThreadName("stratum");
```
Comment (AndreaLanfranchi) [inline libpoolprotocols/stratum/EthStratumClient.cpp]: Please leave setThreadName only immediately before cwarn.
No need to force the name if we have nothing to write in output.

File: libpoolprotocols/stratum/EthStratumClient.cpp [hunk 2]
```diff
@@ -319,6 +318,8 @@ void EthStratumClient::reset_work_timeout()
 
 void EthStratumClient::start_connect()
 {
+    dev::setThreadName("stratum");
```
Comment (AndreaLanfranchi) [inline libpoolprotocols/stratum/EthStratumClient.cpp]: Same as comment above.

**Comment by lookfirst** [comment]
Ok, will submit a new PR.

**Comment by StefanOberhumer** [comment]
@lookfirst 
If you have to adjust the commits in a PR you don't have to delete your branch and create a new one .
You can make more commits into your branch and then 
* rebase your branch interactive against master using (`git rebase -i master`) 
(select which commits to include, squashing, deleting ....)
* push it again into your repo by forcing the push (`git push -f`)
The PR will notice the new push of your branch !

**Comment by lookfirst** [comment]
@StefanOberhumer Yep, thanks for the info! I was just being lazy and didn't feel like rebasing. Easy enough for me to just cherry pick the commits using sourcetree and make a new PR, which I've done already. =)

**Pull request closed without merging**